src/DistributedComputations/distributed_fft_based_poisson_solver.jl

```diff
@@ -32,21 +28,17 @@ Return an FFT-based solver for the Poisson equation,
 ∇²φ = b
 ```
 
-on `Distributed`itectures.
+for `Distributed` architectures.
 
 Supported configurations
 ========================
 
-We support two "modes":
-
-  1. Vertical pencil decompositions: two-dimensional decompositions in ``(x, y)``
-     for three dimensional problems that satisfy either `Nz > Rx` or `Nz > Ry`.
+1. Three dimensional configurations with two-dimensional decompositions in ``(x, y)`` 
+where `Ny ≥ Rx` and `Ny % Rx = 0`, and `Nz ≥ Ry` and `Nz % Ry = 0`.
```

Comment:
```suggestion
1. Three dimensional grids with decompositions in `x` and `y` such that
the `z` direction is continuous, `Ny ≥ Rx` and `Ny % Rx = 0`, and `Nz ≥ Ry` and `Nz % Ry = 0`.
```

Reply:
Also can you explain what `Rx` is before you explain the configurations?

Reply:
Also I'm assuming `Ny` here refers to `global_grid`, correct? Either way, I think making it clear would be better. Something like

```
 `global_grid.Ny ≥ Rx` and `global_grid.Ny % Rx = 0`, and `Nz ≥ Ry` and `Nz % Ry = 0`.
```